src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Helper/ResultHelper.cs

```diff
@@ -0,0 +1,181 @@
+        /// </summary>
+        /// <param name="list">The original result list to convert.</param>
+        /// <param name="iconPath">The path to the icon of each entry.</param>
+        /// <returns>A list with <see cref="Result"/>.</returns>
+        internal static List<Result> GetResultList(
+            in IEnumerable<IWindowsSetting> list,
+            string query,
+            in string iconPath)
+        {
+            var resultList = new List<Result>(list.Count());
+
+            foreach (var entry in list)
+            {
+                var type = entry.Command.StartsWith("ms-settings", StringComparison.InvariantCultureIgnoreCase)
+                    ? Resources.SettingsApp
+                    : Resources.ControlPanel;
+
+                var result = new Result
+                {
+                    Action = (_) => DoOpenSettingsAction(entry),
+                    IcoPath = iconPath,
+                    SubTitle = $"{Resources.Area}: {entry.Area} - {Resources.Type}: {type}",
+                    Title = entry.Name,
+                    ContextData = entry,
+                };
+
+                AddOptionalToolTip(entry, result);
+
+                resultList.Add(result);
+            }
+
+            SetScores(resultList, query);
+
+            return resultList;
+        }
+
+        /// <summary>
+        /// Add a tool-tip to the given <see cref="Result"/>, based o the given <see cref="IWindowsSetting"/>.
+        /// </summary>
+        /// <param name="entry">The <see cref="IWindowsSetting"/> that contain informations for the tool-tip.</param>
+        /// <param name="result">The <see cref="Result"/> that need a tool-tip.</param>
+        private static void AddOptionalToolTip(IWindowsSetting entry, Result result)
+        {
+            var toolTipText = new StringBuilder();
+
+            if (entry.AltNames != null && entry.AltNames.Any())
+            {
+                var altList = entry.AltNames.Aggregate((current, next) => $"{current}, {next}");
+
+                toolTipText.Append(Resources.AlternativeName);
+                toolTipText.Append(": ");
+                toolTipText.AppendLine(altList);
+            }
+
+            toolTipText.Append(Resources.Command);
+            toolTipText.Append(": ");
+            toolTipText.Append(entry.Command);
+
+            if (!string.IsNullOrEmpty(entry.Note))
+            {
+                toolTipText.AppendLine(string.Empty);
+                toolTipText.AppendLine(string.Empty);
+                toolTipText.Append(Resources.Note);
+                toolTipText.Append(": ");
+                toolTipText.Append(entry.Note);
+            }
+
+            var type = entry.Command.StartsWith("ms-settings", StringComparison.InvariantCultureIgnoreCase)
+                ? Resources.SettingsApp
+                : Resources.ControlPanel;
+
+            result.ToolTipData = new ToolTipData(type, toolTipText.ToString());
+        }
```

Comment:
**I suggest to change tool tip like this:**

![image](https://user-images.githubusercontent.com/61519853/116818911-009d6780-ab6e-11eb-9d04-fb688d20d6e8.png)

![image](https://user-images.githubusercontent.com/61519853/116818926-11e67400-ab6e-11eb-9995-5aaa250c1f57.png)

```suggestion
       private static void AddOptionalToolTip(IWindowsSetting entry, Result result)
        {
            var toolTipText = new StringBuilder();

            var type = entry.Command.StartsWith("ms-settings", StringComparison.InvariantCultureIgnoreCase)
                ? Resources.SettingsApp
                : Resources.ControlPanel;

            toolTipText.Append(Resources.Application);
            toolTipText.Append(": ");
            toolTipText.AppendLine(type);

            toolTipText.Append(Resources.Area);
            toolTipText.Append(": ");
            toolTipText.AppendLine(entry.Area);

            if (entry.AltNames != null && entry.AltNames.Any())
            {
                var altList = entry.AltNames.Aggregate((current, next) => $"{current}, {next}");

                toolTipText.Append(Resources.AlternativeName);
                toolTipText.Append(": ");
                toolTipText.AppendLine(altList);
            }

            toolTipText.Append(Resources.Command);
            toolTipText.Append(": ");
            toolTipText.Append(entry.Command);

            if (!string.IsNullOrEmpty(entry.Note))
            {
                toolTipText.AppendLine(string.Empty);
                toolTipText.AppendLine(string.Empty);
                toolTipText.Append(Resources.Note);
                toolTipText.Append(": ");
                toolTipText.Append(entry.Note);
            }

            result.ToolTipData = new ToolTipData(entry.Name, toolTipText.ToString());
        }
```

Reply:
Added